pkg/network/protocols/postgres/protocol.go

```diff
@@ -200,7 +200,11 @@ func (p *protocol) processPostgres(events []EbpfEvent) {
 		tx := &events[i]
 		eventWrapper := NewEventWrapper(tx)
 		p.statskeeper.Process(eventWrapper)
-		p.telemetry.Count(tx, eventWrapper)
+		if p.cfg.MaxPostgresTelemetryBuffer > 0 {
```

Comment:
can you elaborate on the condition here?

Reply:
Actually, this is a bug. I have changed it to check if the cfg value is greater than the default value, and if so, I am using the cfg value.